### PR TITLE
Cert no write #117

### PR DESCRIPTION
**Pull request by AndrewCopeland**
If template `StoreCertificate` is `false` then do not store certificate in the conjur backend.
With this PR we can now run the PKI service directly against a follower. Before on startup the PKI service would load an initialization policy. However writes are not possible against a follower. The PKI service will hit the conjur appliance `/info` endpoint and retrieve the role of the appliance. If the role of the appliance is `follower` then the init policy load will not occur. If the role of the appliance is `master` then the init policy load will occur.

### CODE REVIEW

File: internal/backend/conjur/templates.go [hunk 1]
```diff
@@ -82,7 +82,9 @@ func (c StorageBackend) ListTemplates() ([]string, error) {
 func (c StorageBackend) GetTemplate(templateName string) (types.Template, error) {
 	variableID := c.getTemplateVariableID(templateName)
 	templateJSON, err := c.client.RetrieveSecret(variableID)
-	template := &types.Template{}
+	template := &types.Template{
+		StoreCertificate: true,
```
Comment (infamousjoeg) [inline internal/backend/conjur/templates.go:86]: Is this to set the default bool value to true?  Just wanted to confirm...

File: pkg/pkiaas/api/template_controller.go [hunk 2]
```diff
@@ -41,7 +41,8 @@ func CreateTemplateHandler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, httpErr.JSON(), httpErr.HTTPResponse)
 		return
 	}
-	var newTemplate types.Template
+
+	newTemplate := &types.Template{StoreCertificate: true}
```
Comment (infamousjoeg) [inline pkg/pkiaas/api/template_controller.go:45]: Here it is again...

File: internal/backend/conjur/templates.go (hunk 1 above)
Comment (AndrewCopeland) [reply internal/backend/conjur/templates.go:86]: It defaults to true, and if false is provided is overwrites the true.

File: pkg/pkiaas/api/template_controller.go (hunk 2 above)
Comment (AndrewCopeland) [reply pkg/pkiaas/api/template_controller.go:45]: It defaults to true. If false is provided it overrides the default.

**Pull request merged**